recipes/bzip3/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+import functools
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+class BZip3Conan(ConanFile):
+    name = "bzip3"
+    description = "A better and stronger spiritual successor to BZip2."
+    license = "LGPL-3.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/kspalaiologos/bzip3"
+    topics = ("bzip2", "lzma", "compression")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_thread": [True, False],
+        "with_util": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_thread": True,
+        "with_util": False,
+    }
+    generators = "cmake"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def export_sources(self):
+        self.copy("CMakeLists.txt")
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            self.copy(patch["patch_file"])
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.cppstd
+        del self.settings.compiler.libcxx
+        if self.settings.os not in ["Linux", "FreeBSD"]:
+            del self.options.with_thread
```

Comment:
- should be in `config_options()`
- why only Linux & FreeBSD? Should be deleted for Windows only (and actually it should work for MinGW with pthread)